utils/ANLEra_auto_working.cfg

```diff
@@ -327,3 +327,95 @@
         [/redraw]
     [/event]
 #enddef
+
+# ------------------------------------------------------
+# Automatic Hunting
+# ------------------------------------------------------
+
+#define AUTO_HUNTING TYPES
+    [event]
+        name=side turn
+        first_time_only=no
+
+        [store_unit]
+            [filter]
+                type={TYPES}
+                side=$side_number
+                canrecruit=no
+                [filter_location]
+                    terrain=*^F**
+				[/filter_location]
+            [/filter]
+            variable=hunters
+            kill=no
+        [/store_unit]
+
+        [set_variable]
+            name=sound
+            value=no
+        [/set_variable]
+        [foreach]
+            array=hunters
+            [do]
+                [set_variable]
+                    name=credited_side
+                    value=$this_item.side
+                [/set_variable]
+                [set_variable]
+                    name=amount
+                    value="$(($this_item.level + 1) * 2)"
```

Comment:
Quite much, 8 gold for an lvl 3, while a White Mage gives 3 research points.
2 gold per turn for a Woodsman (costs 10g to recruit)
3 gold per 2 turns for a Peasant (costs 8g to recruit), (and a third turn in case you want to reuse the terrain later, though you have probably researched at that point)
I had $(($this_item.level + 1)) in mind, but it may be a bit cheap indeed. But he has as well 60% while being in forests, while the Peasant works on 40% farmland, and his offensive potential is better.
(Didn't compare with mines)

For the Outlaws it's a bigger difference, with 6g vs 10g initial price. Didn't consider the details for them yet, maybe $(($this_item.level +2))

Reply:
Yeah, I was thinking it might be a little much at higher levels